### DISCUSSION TITLE
cron job for drop usage

### DISCUSSION THREAD

**Discussion by Fayozjon** [bpuig/laravel-subby#117] [Q&A] [answered]
Hello can you provide TASKS for CRON JOB for example restarting limits of each Plan Feauture USAGE count

For example i have plan that have Feature resettable_period = 1 and resettable_interval = day
so i need to reset every day for example at 00:00 usages of subscribers

**Comment by bpuig** [comment, marked as answer]
Hello!

It resets automatically when a new use is made via `recordFeatureUsage`. If you want to display usage, you can use `getFeatureUsage`. Take a look at [this](https://bpuig.github.io/laravel-subby/v5.x/models/plan-subscription-model.html#subscription-feature-usage)

**Comment by Fayozjon** [comment]
Thank you very much for answers 
I Appreciate this!
